pumps/sql_aggregate.go

```diff
@@ -198,7 +198,7 @@ func (c *SQLAggregatePump) ensureIndex(tableName string, background bool) error
 		c.log.Info("Creating index for table ", tableName, "...")
 		return createIndexFn(c)
 	}
-	c.log.Info(newAggregatedIndexName, " already exists.")
+	c.log.Info(indexName, " already exists.")
```

Comment:
will this be printed every time we run pump and the indexes were already created? If so, this could create a lot of noise

Reply:
thats true, we dont need to print that. And we are already printing errors inside the `createIndexFn` so am removing it